### ISSUE TITLE
Text markup broken on text_entry_dialog (Trac #95)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#99]
See, for example, the superscript in the edit electrode area dialog from dmf_device_controler.py (image attached).

Migrated from http://microfluidics.utoronto.ca/microdrop/ticket/95
```json
{
    "status": "closed", 
    "changetime": "2014-04-17T19:39:43", 
    "description": "See, for example, the superscript in the edit electrode area dialog from dmf_device_controler.py (image attached).", 
    "reporter": "ryan", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1397763583719301", 
    "component": "microdrop", 
    "summary": "Text markup broken on text_entry_dialog", 
    "priority": "minor", 
    "keywords": "gui", 
    "version": "0.1", 
    "time": "2012-04-07T04:21:32", 
    "milestone": "Microdrop 1.0", 
    "owner": "cfobel", 
    "type": "defect"
}
```

**Comment by cfobel** [comment]
Trac update at 20120407T04:21:45: ryan changed attachment from "" to "edit_electrode_area.png"

**Comment by cfobel** [comment]
Trac update at 20120417T04:39:02: 
* christian@fobel.net commented:

> commit 595f1ffdb9181d4e5bddd92f1b7aeb8ac8267a2a
> Author: Christian Fobel <christian@fobel.net>
> Date:   Tue Apr 17 00:21:30 2012 -0400
> 
>     Add use_markup kwarg to utility dialog functions
>     
>     Add use_markup kwarg to utility dialog functions to indicate whether or not the
>     label text should be processed as marked-up text (default: False, i.e., text is
>     not processed as markup).
>     
>     Fixes #99 by specifying use_markup=True for the electrode area dialog.


* christian@fobel.net changed resolution from "" to "fixed"
* christian@fobel.net changed status from "new" to "closed"

**Comment by cfobel** [comment]
Trac update at 20140417T19:39:43: ryan changed component from "" to "microdrop"
